### ISSUE TITLE
[Bug] Wonder does not show stats

### ISSUE THREAD

**Issue by AugustoXDBR** [SiegeEngineers/aoe2techtree#52]
The wonder doesn't seem to get it's metadata right
![Screenshot_3](https://user-images.githubusercontent.com/58894348/81758560-8e350280-9498-11ea-86af-b6d82cccfec8.png)

If it helps with anything, i've made this awfulness, that althogh works, looks horrible: 
```html
         if (meta !== undefined) {
            text = text.replace(/‹cost›/, "Cost: " + cost(meta.Cost));
            let stats = []
            if (text.match(/‹hp›/)) {
                stats.push("HP:&nbsp;" + meta.HP);
                text = text.replace(/‹hp›/, "HP:&nbsp;" + meta.HP);
            }
            if (text.match(/‹attack›/)) {
                stats.push("Attack:&nbsp;" + meta.Attack);
                text = text.replace(/‹attack›/, "Attack:&nbsp;" + meta.Attack);
            }
            if (text.match(/‹(A|a)rmor›/)) {
                stats.push("Armor:&nbsp;" + meta.MeleeArmor);
                text = text.replace(/‹(A|a)rmor›/, "Armor:&nbsp;" + meta.MeleeArmor);
            }
            if (text.match(/‹(P|p)iercearmor›/)) {
                stats.push("Pierce armor:&nbsp;" + meta.PierceArmor);
                text = text.replace(/‹(P|p)iercearmor›/, "Pierce armor:&nbsp;" + meta.PierceArmor);
            }
            if (text.match(/‹garrison›/)) {
                stats.push("Garrison:&nbsp;" + meta.GarrisonCapacity);
                text = text.replace(/‹garrison›/, "Garrison:&nbsp;" + meta.GarrisonCapacity);
            }
            if (text.match(/‹range›/)) {
                stats.push("Range:&nbsp;" + meta.Range);
                text = text.replace(/‹range›/, "Range:&nbsp;" + meta.Range);
            }
```
Good luck and I hope you don't get annoyed at that many bug reports haha

**Comment by HSZemi** [comment]
A wonder does not list upgrades, so the replacement [here](https://github.com/SiegeEngineers/aoe2techtree/blob/master/index.html#L452) fails.

@lalitpatel would you mind having a look at this?

**Comment by lalitpatel** [comment]
@HSZemi I was going through [this page](https://ageofempires.fandom.com/wiki/Wonder_(Age_of_Empires_II)#Further_statistics). It seems Wonders do get affected by upgrades like Town Watch, Masonry etc. Dose it mean that the data in the `data.json` file is incorrect for Wonders?

**Comment by HSZemi** [comment]
That depends on your definition of "incorrect". I would say the "correct" text is the one from the game files, hence the data in `data.json` seems to be correct for Wonders. But sadly, that text is *inconsistent* with the other texts in that it does not contain information about upgrades.

**Comment by HSZemi** [comment]
The issue has been resolved with #53 :tada: 
Thank you for the bug report @AugustoXDBR
